### PR TITLE
migrating vttest_sample python to go

### PR DESCRIPTION
**Pull request by princeparmar**
DO NOT MERGE until after #5799 and #5826 have been merged.
!!! this pr is having the changes of web-test and  xb-recovery !!!

vttest_sample flow...
used vtcombo for cluster.
inserted multiple value by grpc connection with one tablet (first in ks1:-80@master and then ks1:80-@master)
then validated the inserted values in respecte shards
created one redirected ks.
validated inserted values in replica of the redirected shard and rdonly of the ks1
executed tablet list cmd in vtctlclient and validated output.

### CODE REVIEW

**Comment by deepthi** [comment]
Looks fine, but we need to finish reviewing and merging of #5826 first.

**Comment by morgo** [comment]
The other PRs have now merged, so we should be able to merge this now after fixing conflicts.

File: test.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ var (
 	runCount       = flag.Int("runs", 1, "run each test this many times")
 	retryMax       = flag.Int("retry", 3, "max number of retries, to detect flaky tests")
 	logPass        = flag.Bool("log-pass", false, "log test output even if it passes")
-	timeout        = flag.Duration("timeout", 10*time.Minute, "timeout for each test")
+	timeout        = flag.Duration("timeout", 15*time.Minute, "timeout for each test")
```
Comment (deepthi) [inline test.go]: Why did we need to increase this?

File: .github/workflows/cluster_endtoend.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        name: [11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24]
+        name: [11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 26]
```
Comment (deepthi) [inline .github/workflows/cluster_endtoend.yml]: Shard 25?

Comment (princeparmar) [reply .github/workflows/cluster_endtoend.yml]: shard 25 is running using saperate yaml ( .github/workflows/cluster_vtctl_web.yml )

Comment (ajeetj) [reply .github/workflows/cluster_endtoend.yml]: It was the moment of testing and validating. I think we should move that test to shard 10. 

@princeparmar

**Pull request merged**